src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -364,6 +364,7 @@ well as menu structures (for main menu and popup menus).
             <cmd refid="importDatasetFromFile"/>
             <cmd refid="importDatasetFromURL"/>
             <cmd refid="importDatasetFromCSV"/>
+            <cmd refid="importDatasetFromSAV"/>
```

Comment:
I think there should be a separator after import from text. Final menu might look something like this:

```
Import Dataset from Text...
-------------------------------------------
Import Dataset from Excel...
-------------------------------------------
Import Dataset from SPSS...
Import Dataset from SAS...
Import Dataset from Stata...
```